File: SecondYear/chaudharyaryanpanwar_AryanPanwar_2226cs1053_2/Week_1/Pattern4.cpp
class Solution {
    public :
    void printTriangle(int n){
        //code here
         for (int i = 1 ;i <= n ; i++ ){
            for (int j = 1 ; j< i+1 ; j++){
                cout<<i<<" ";
            }
            cout<< endl;
        }
    }
};

File: SecondYear/Vaishnavi0-0_VaishnaviSingh_2226cseml1114_2/week_1/pattern4.java
import java.util.Scanner;

public class pattern4 {

    public static void main(String[] args) {
        try (Scanner read = new Scanner(System.in)) {
            int n = read.nextInt();
            printTriangle(n);
        }
    }

    public static void printTriangle(int n) {
        // code here
        for (int i = 1; i <= n; i++) {
            for (int j = 1; j <= i; j++) {
                System.out.print(i + " ");
            }
            System.out.println();
        }
    }

}
